async-nats/tests/jetstream_tests.rs

```diff
@@ -1280,6 +1281,73 @@ mod jetstream {
         assert_eq!(info.num_ack_pending, 8);
     }
 
+    #[tokio::test]
+    #[cfg_attr(target_os = "windows", ignore)]
```

Comment:
I take it that Windows reconnect remain flaky?

Reply:
Hm, i actually changed the reconnections, allowing reset of server, but I would limit windows tests where feasible. Those windows gha runners are so slow they have weird impact on nats-server and connection.